benchmarks/benchmarks/bench_function_base.py

```diff
@@ -98,44 +98,122 @@ def time_select_larger(self):
 class Sort(Benchmark):
     params = [
         ['quick', 'merge', 'heap'],
```

Comment:
Might as well use stable instead of merge.

Reply:
NVM, 'stable' is too new for testing older NumPy versions. Maybe a note that in NumPy 1.17 it can be any of several stable sorts.